mm/ktsan/trace.c

```diff
@@ -66,6 +66,7 @@ void kt_trace_add_event(kt_thr_t *thr, kt_event_type_t type, uptr_t addr)
 		kt_trace_switch(trace, clock);
 
 	event.type = (int)type;
+	event.cpu_id = smp_processor_id();
```

Comment:
This adds cost to memory access handling. I am not sure it worth it.
Can we use the trace power to eliminate that cost? That is, write an event with current cpu to trace when a thread is scheduled, then replay the trace and obtain the cpu this way?